Misc/NEWS.d/next/Library/2019-10-30-15-31-09.bpo-38641.HrTL9k.rst

```diff
@@ -0,0 +1 @@
+Added starred expressions support to ``return`` and ``yield`` statements.
```

Comment:
Can you specify this is a bug related to `lib2to3` here, also you can add a `Patch by <name>.` attribution at the end of news.

Reply:
Updated, thank you!